basculeLogging.go

```diff
@@ -3,15 +3,17 @@ package main
 import (
 	"context"
 	"net/http"
+	"os"
 	"strings"
 
 	"github.com/go-kit/log"
 	"github.com/xmidt-org/candlelight"
-
-	// nolint:staticcheck
-	"github.com/xmidt-org/webpa-common/v2/logging"
 )
 
+type contextKey uint32
```

Comment:
Using context keys in this way is overly verbose.  The standard pattern in `golang` is to leverage the fact that comparison is based on type first, value second:

```
type contextKey struct{}

context.WithValue(parentCtx, contextKey{}, logger)
```

However, see my note below about using `sallust` instead.